lib/htty/cli/commands/path_set.rb

```diff
@@ -40,7 +40,9 @@ def self.see_also_commands
   def perform
     add_request_if_new do |request|
       self.class.notify_if_cookies_cleared request do
-        request.path_set(*escape_or_warn_of_escape_sequences(arguments))
+        request.path_set *(arguments.map do |argument|
+          escape_or_warn_of_escape_sequences(argument.split('/')).join('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.